### DISCUSSION TITLE
temporary password length

### DISCUSSION THREAD

**Discussion by egs2003** [rustdesk/rustdesk#3615] [Q&A]
I would like to change the default length of the temporary password to 10 in my personal version. But I can't find the code. Can anyone help me?

**Comment by ghost** [comment]
In your config file "RustDesk2.toml", you gotta add a key like that:
![image](https://user-images.githubusercontent.com/107928926/224937047-423ba9b1-d810-4477-a128-3174c1a6428c.png)

This conf file is located at :
- Windows : "C:\Windows\ServiceProfiles\LocalService\AppData\Roaming\RustDesk\config"
- Ubuntu : /home/$USER/.config/Rustdesk or /root/.config/rustdesk depending on wether you wanna apply it to all user (/root) or one user (/home/$USER)

Then restart service. If it doesn't work you probably have to restart the machine.
